src/breadcrumb-group/implementation.tsx

```diff
@@ -106,10 +126,45 @@ export function BreadcrumbGroupImplementation<T extends BreadcrumbGroupProps.Ite
     checkSafeUrl('BreadcrumbGroup', item.href);
   }
   const baseProps = getBaseProps(props);
-  const isMobile = useMobile();
+  const [navWidth, navRef] = useContainerQuery<number>(rect => rect.contentBoxWidth);
```

Comment:
This should be `rect.borderBoxWidth` to incorporate inner paddings, if there are any